app/src/main/java/de/tum/in/tumcampusapp/tumonline/TUMOnlineRequestFetchListener.java

```diff
@@ -34,4 +34,9 @@
 	 */
 	void onFetchError(String errorReason);
 
+	/**
+	 * Called if there was no error accessing tum online but there are no results to show (no entries)
+	 */
+	void onNoDataToShow();
```

Comment:
Great idea, this is probably also useful for a couple of other Activities, where we currently just show generic error messages...